packages/authentication/integration-tests/__tests__/services/providers/username-password.spec.ts

```diff
@@ -91,6 +92,7 @@ describe("AuthenticationModuleService - AuthProvider", () => {
 
       const res = await service.authenticate("usernamePassword", {
```

Comment:
would be nice to rename this to something like `userpass` or `user-pass`

Reply:
Or use `local`. Think people will find that familiar from passportjs

Reply:
rename in #6251